hta/configs/logging.config

```diff
@@ -12,7 +12,7 @@ level=INFO
 handlers=consoleHandler
 
 [logger_hta]
-level=INFO
+level=DEBUG
```

Comment:
@rzdao Is this change for testing purposes or are you planning to commit this change?

Reply:
This is just for testing purpose. Changed back to INFO